Makefile.defs

```diff
@@ -192,9 +192,9 @@ GO_CLEAN = $(GO) clean $(GO_CLEAN_FLAGS)
 GO_VET = $(GO) vet
 GO_LIST = $(GO) list
 
-HELM_TOOLBOX_VERSION ?= "v1.0.2"
-HELM_TOOLBOX_SHA ?= "c5a728e61e1f04580c1a0074f9bf3ab96f7beca687385878b403bb0e52163d34"
-HELM_TOOLBOX_IMAGE ?= "quay.io/cilium/helm-toolbox:$(HELM_TOOLBOX_VERSION)@sha256:$(HELM_TOOLBOX_SHA)"
+HELM_TOOLBOX_VERSION ?= "v1.1.0"
+HELM_TOOLBOX_SHA ?= "93ec4d691c634b10a89f1db5d88faaae34318465e81c3bff576256db4aa264c2"
+HELM_TOOLBOX_IMAGE ?= "quay.io/geraldpape/helm-toolbox:$(HELM_TOOLBOX_VERSION)@sha256:$(HELM_TOOLBOX_SHA)"
```

Comment:
We can coordinate this with https://github.com/cilium/helm-toolbox/pull/3. Once that PR is merged, we can tag a new cilium/helm-toolbox and update these references.

Reply:
Yes, that would be awesome

Reply:
I'm trying this out locally via the PR, and a bunch of the flags like `--omit-additional-properties --omit-default --omit-description --omit-title --omit-required-properties` are not available. Are those net-new features?

This is `--help` with version 0.9.0:

```
helm-schema automatically generates a jsonschema file for helm charts from values files

Usage:
  helm-schema [flags]

Flags:
  -c, --chart-search-root string       directory to search recursively within for charts (default ".")
  -x, --dont-strip-helm-docs-prefix    Disable the removal of the helm-docs prefix (--)
  -d, --dry-run                        don't actually create files just print to stdout passed
  -h, --help                           help for helm-schema
  -s, --keep-full-comment              Keep the whole leading comment (default: cut at empty line)
  -l, --log-level string               Level of logs that should printed, one of (panic, fatal, error, warning, info, debug, trace) (default "info")
  -n, --no-dependencies                don't analyze dependencies
  -o, --output-file string             jsonschema file path relative to each chart directory to which jsonschema will be written (default "values.schema.json")
  -k, --skip-auto-generation strings   comma separated list of fields to skip from being created by default (possible: title, description, required, default, additionalProperties)
  -f, --value-files strings            filenames to check for chart values (default [values.yaml])
  -v, --version                        version for helm-schema
```

Reply:
Yeah this changed. Try with `--skip-auto-generation title,description,required,default,additionalProperties`

Reply:
Pushed an update to the Makefile now

Reply:
I just tagged `quay.io/cilium/helm-toolbox:v1.1.0` at sha `961693f182b9b456ed90e5274ac5df81e4af4343104e252666959cdf9570ce9e`.